app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/SessionFeedbackFragment.java

```diff
@@ -100,11 +100,15 @@ public void onClickSubmitFeedback() {
 
     public void onSubmitSuccess() {
         // TODO: show success action
+        if (isDetached())
```

Comment:
Let me ask you one thing.
Can we clear `Disposable` when Fragment is detached?
I mean like this.

```java
     @Override
     public void onClickSubmitFeedback() {
         SessionFeedback sessionFeedback = new SessionFeedback(sessionId);
         disposable = viewModel.submitSessionFeedback(sessionFeedback)
                 .subscribeOn(Schedulers.io())
                 .observeOn(AndroidSchedulers.mainThread())
                 .subscribe(success -> onSubmitSuccess(), failure -> onSubmitFailure());

        compositeDisposable.add(disposable);
     }

    @Override
    public void onDetach() {
        super.onDetach();
        compositeDisposable.clear();
    }
```